docs/source/user.rst

```diff
@@ -0,0 +1,199 @@
+..  _user:
+
+Here the simple examples cover the basics of creating messaging applications using kiwipy over RabbitMQ.
+
+The parameters section will focus on some of the important parameters that initialize the communicator and their meaning.
```

Comment:
Also this we could move under the corresponding headline (the sections will anyhow be shown in the sidebar